assets/scss/_pageHead.scss

```diff
@@ -0,0 +1,10 @@
+@include responsiveBase() {
+	.pageHead {
+		padding-bottom: $base*2 - $base;
+	}
+
+	.pageActionButton {
```

Comment:
Is there a reason why this style is include here rather `_pageActionButton.scss`?